pkg/compactor/blocks_cleaner.go

```diff
@@ -47,14 +50,16 @@ type BlocksCleaner struct {
 	bucketClient objstore.InstrumentedBucket
 	usersScanner *cortex_tsdb.UsersScanner
 
+	ringLifecyclerID string
```

Comment:
Is this related to this PR? I cant find where it is being used. Only we setting it. Maybe it was mixed in the split of PRs

Reply:
Removed